test/dev-requirements.txt

```diff
@@ -0,0 +1,8 @@
+beautifulsoup4>=4.9.1
+behave>=1.2.6
+selenium>=4.27.1
+git+https://github.com/duttonw/behaving.git@main#egg=behaving
```

Comment:
Was looking at splitting behaving testing from ckan normal installs don't get the extra bits.